aksel.nav.no/website/app/dev/_ui/Avatar.module.css

```diff
@@ -0,0 +1,36 @@
+.avatarList {
+  display: flex;
+}
+
+.avatarItem {
+  &:not(:first-child) {
+    margin-left: calc(-1 * var(--ax-space-8));
+    border-radius: var(--ax-radius-full);
+    position: relative;
+
+    &::after {
+      inset: 0;
+      border-radius: inherit;
+      box-shadow: inset 0 0 0 1px var(--ax-border-subtleA);
+      content: "";
+      position: absolute;
+    }
+
+    & .avatarImage {
+      box-shadow: 0 0 0 2px var(--ax-bg-default);
+      background-clip: content-box;
+      position: relative;
+      border-radius: var(--ax-radius-full);
+      overflow: hidden;
+    }
+  }
+}
+
+.avatarName {
+  margin-block: var(--ax-space-1) 0;
+  margin-inline: var(--ax-space-2) 0;
+}
```

Comment:
Is this needed? Can `gap` on HStack solve it?

Reply:
it was a very subtle spacing to be added to the top and left to position the team name "just right" in relation to the Avatar image... this is yet again due to our font 😢, since I have an align="center", but it's just a pixel off.

I could maybe replace the inline spacing by adding more gap though.

Reply:
oh wait... I keep forgetting `marginInline` and `marginBlock` on our layout components 😳, with asChild they can go onto anything.